OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnDeployWithCharge.cs

```diff
@@ -46,6 +46,9 @@ public class GrantConditionOnDeployWithChargeInfo : PausableConditionalTraitInfo
 		[Desc("Can " + nameof(DeployedCondition) + " be canceled by followup deploy order?")]
 		public readonly bool CanCancelCondition = false;
 
+		[Desc("Can the actor deploy while moving?")]
+		public readonly bool CanDeployWhileMoving = false;
```

Comment:
```suggestion
		public readonly bool CancelsCurrentOrders = false;
```
To be more generic as this doesn't specifically apply to movement.

Reply:
your suggestion is plain wrong though, we never cancel orders

Reply:
and this does specifically apply to movement

Reply:
renamed the trait to ToggleChargedConditionOnOrder